synapse/storage/databases/main/pusher.py

```diff
@@ -196,27 +196,6 @@ async def get_if_user_has_pusher(self, user_id: str):
         # This only exists for the cachedList decorator
         raise NotImplementedError()
 
-    @cachedList(
-        cached_method_name="get_if_user_has_pusher",
-        list_name="user_ids",
-        num_args=1,
-    )
-    async def get_if_users_have_pushers(
```

Comment:
This was unused since #8059 as far as I can tell.

Reply:
The signature of this is now "fine" that it could stay and mypy won't complain, but if it is unused...I think we should remove it?